cli/utils/utils.go

```diff
@@ -43,3 +43,24 @@ func ValidateOutputPathFileMode(fileMode os.FileMode) error {
 	}
 	return nil
 }
+
+// GetHomeDir gets the home directory. If we're in CI, then it's the CWD, otherwise assume it's
+// /home/seluser
+func GetHomeDir() (string, error) {
+	if (os.Getenv("CI") == "") {
+		return "/home/seluser", nil
+	}
+	workingDir, err := os.Getwd()
+	if (err != nil) {
+		return "", err
+	}
+	return workingDir, nil
+}
+
+// GetConfigFile config yaml. If we're in CI, then it's config.yaml, otherwise it's config-local.yaml
+func GetConfigFile () string {
```

Comment:
Hm. This is too ambiguous. `utils.GetConfigFile()`: As a caller, I'd have no idea what this does. There's the saucectl config file, the runner config file and perhaps a dime and dozen other.

If the logic is runner specific, then I'd say the functionality should also move closer to the runner.
We should kill the entire `utils` package as soon as possible.